docs/docs/dev_docs/getting_started/updating.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Updating
+---
+
+There are 4 components whose versions need to be kept compatible:
+1. Aztec Sandbox,
+2. Aztec CLI,
+3. Noir compiler `nargo`,
+4. Noir framework for Aztec contracts `aztec.nr`.
+
+## Updating Aztec Sandbox
+To update the sandbox to the latest version, simply run the curl command we used for installation again:
+```shell
+/bin/bash -c "$(curl -fsSL 'https://sandbox.aztec.network')"
+```
+
+It will download and start the latest version of sandbox.
+
+If you would like to use a fixed version of the sandbox, you can export the `SANDBOX_VERSION` environmental variable.
+If you are unsure what version to use go to [aztec-packages repository](https://github.com/AztecProtocol/aztec-packages/releases) and choose the `aztec-packages` release based on the changelog.
+
+Then set the `SANDBOX_VERSION` environmental variable to the version you want to use. E.g.:
+```shell
+export SANDBOX_VERSION=0.7.7
+```
+
+Now when you run the curl command it will use the version you specified.
+To verify that it's the case check the console output of the curl command.
+You should see the following line:
+```
+Aztec Sandbox 0.7.7 is now ready for use!
+```
+
+Alternatively you can open a new terminal and use aztec-cli to get the version.
+
+#include_code node-info yarn-project/end-to-end/src/cli_docs_sandbox.test.ts bash
+
+The sandbox version should be the same as the one we chose by setting the `SANDBOX_VERSION` environmental variable.
+
+## Updating Aztec CLI
+If the latest version was used when updating the sandbox then we can simply run the following command to update the CLI:
+```shell
+npm install -g @aztec/cli
+```
+
+If a specific version was set for the sandbox then we need to install the CLI with the same version:
+```shell
+npm install -g @aztec/cli@REPLACE_WITH_SANDBOX_VERSION
+```
+
+E.g.:
+```shell
+npm install -g @aztec/cli@0.7.7
+```
+
+## Updating Noir compiler
+Now we need to update the Noir compiler `nargo` to the version compatible with the sandbox.
+Use `aztec-cli` to get it:
+#include_code node-info yarn-project/end-to-end/src/cli_docs_sandbox.test.ts bash
+
+Then we install the `Compatible Nargo Version` with (replace `COMPATIBLE_NARGO_VERSION` with the version from the previous command):
+```shell
+noirup -v COMPATIBLE_NARGO_VERSION
```

Comment:
Let's load the latest compatible version from the `NoirVersion` component. Note that you'll need to explicitly use a `CodeBlock`, since the triple backtick will not evaluate the react component.

https://github.com/AztecProtocol/aztec-packages/blob/18fe88aa63e9a93f4f62789e94047edf33620bfa/docs/src/components/InstallNargoInstructions/index.js#L5-L12

Reply:
In this case this value is related to the one from get-node-info:
<img width="573" alt="image" src="https://github.com/AztecProtocol/aztec-packages/assets/13470840/8073fbbe-58aa-4beb-a812-cf2bd2d28078">

So we would need to inject the version in both places. We can't easily interpolate the noir version in the get-node-info output because that is the hardcoded CLI output imported from the test.

We could create an alternative to the include_code macro which would import code and replace a string (e.g. in this case it would replace NARGO_VERSION string with real value).  But TBH not sure if it's worth the hassle because in this case people might want to update to a specific version and not necessarily to the newest one so not having here the newest version doesn't look like that big of a deal to me.

WDYT?

Reply:
Hah, it's a very good point. Yeah, let's leave this as is then. Not worth the hassle.